docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -362,17 +386,17 @@ _build/validator/rel/miner/bin/miner start
 One last step. When building from source, you'll need to manually load the genesis block once your miner is deployed and running. To do this, run the following:
 
 ```
-wget https://snapshots.helium.wtf/genesis.testnet
+wget https://snapshots.helium.wtf/genesis.mainnet
 miner genesis load </absolute/path/to/genesis/block>
 ```
 
 Note that in the above `miner` command, the `miner` binary here is nested within `_build/validator/rel/miner/bin/miner`.
 
 ## Stake Tokens to Your Validator
 
-Now that your Validator node is running, the final step in the process is to formally stake TNT to your Validator. As part of the staking process the Validator address needs to both be in the staking transaction and sign the transaction. After a wallet stakes a validator node, the wallet becomes that node’s owner, has control over that validator node, and receives rewards.
+Now that your Validator node is running, the final step in the process is to formally stake HNT to your Validator. As part of the staking process the Validator address needs to both be in the staking transaction and sign the transaction. After a wallet stakes a validator node, the wallet becomes that node’s owner, has control over that validator node, and receives rewards.
```

Comment:
> Validator address needs to both be in the staking transaction and sign the transaction. After a wallet stakes a validator node, the wallet becomes that node’s owner, has control over that validator node, and receives rewards.
 
 This  "and sign the transaction" isn't clear to me, the Validator address needs to sign the transaction? Do you mean:
 ```
 ...the validator address needs to be in the staking transaction and the transaction needs to be signed
 ```
 ?

Reply:
This is old carryover language from the previous testnet doc. I agree that it is poorly worded. I'll take a stab at rewriting the entire section so that its more relevant and flows better.